traitsui/testing/tester/ui_wrapper.py

```diff
@@ -132,14 +132,19 @@ def locate(self, location):
         Parameters
         ----------
         location : any
-            Any location type supported by the current target.
+            An instance of a location type supported by the current target.
+
```

Comment:
nit: do you want to add an example here like you did with the docstring for interaction?

Reply:
Ah yes, that's a good idea.